neuralprophet/time_net.py

```diff
@@ -170,7 +170,7 @@ def __init__(
             self.config_events = None
             self.config_holidays = None
 
-            # Autoregression
+        # Autoregression
         self.n_lags = n_lags
         self.num_hidden_layers = num_hidden_layers
         self.d_hidden = n_lags + n_forecasts if d_hidden is None else d_hidden
```

Comment:
let's make this:
if d_hidden is None:
    self.d_hidden = max(4, (n_lags + n_forecasts) /(2.0* num_hidden_layers)
else:
    self.d_hidden = d_hidden

Reply:
@ourownstory we have two problems with this suggestion: (1) we can have a 0 division when num_hidden_layers is not set (default = 0) and (2) we could have a max (4, float) , in other words, we could have, for instance, self.d_hidden = 4.5. 
I suggest this:
max(4, round((n_lags + n_forecasts)/(2.0*(num_hidden_layers+1)))